src/components/sidebar/index.tsx

```diff
@@ -73,7 +73,11 @@ const Sidebar = ({
 					[s['filterInputWrapper--mobile']]: shouldRenderMobileControls,
 				})}
 			>
-				<SidebarFilterInput value={filterValue} onChange={setFilterValue} />
+				<FilterInput
+					value={filterValue}
+					onChange={setFilterValue}
+					placeholder="Filter sidebar"
+				/>
```

Comment:
This component was abstracted out for re-use, just adjusting the original usage here.